src/app/public/modules/tabs/tabset-buttons-display-mode.ts

```diff
@@ -0,0 +1 @@
+export type SkyTabsetButtonsDisplayMode = 'dropdown' | 'tabs';
```

Comment:
Needs an `@internal` decorator.

Reply:
This is actually a public type (before it was listed as a `string`, this will make the value more accessible to our consumers).